go-selinux/selinux_linux.go

```diff
@@ -289,7 +289,10 @@ func SetFileLabel(fpath string, label string) error {
 	if fpath == "" {
 		return ErrEmptyPath
 	}
-	return lsetxattr(fpath, xattrNameSelinux, []byte(label), 0)
+	if err := lsetxattr(fpath, xattrNameSelinux, []byte(label), 0); err != nil {
+		return errors.Wrap(err, "Failed to set file label on %s")
```

Comment:
It is missing the argument `fpath`.

Should the error start with a lowercase letter?

Reply:
> Should the error start with a lowercase letter?

By convention yes. Lowercase makes it easier to read when being wrapped. Only logs should start with upper case.

Reply:
Fixed

Reply:
don't see a new version yet, was it pushed?

Reply:
It is there now.